### PR TITLE
Initial release of track 2 module `agrifood`

### PR DESCRIPTION
**Pull request by ArcturusZhang**
The service team is requesting a renaming from `agfood` to `agrifood`. Therefore this PR adds the CIs for the new coming RP `agrifood`

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.
 
[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/master/CHANGELOG.md

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Didn't we already release a module with the old name?

**Comment by ArcturusZhang** [comment]
> Didn't we already release a module with the old name?

Yes, now the case is that the service team wants to change the module name - therefore I am releasing a new one and will delete the old one

**Comment by jhendrixMSFT** [comment]
You can't delete the old one, it will live "forever".  We can, however, ask Google to remove the old one from the module index.

**Comment by ArcturusZhang** [comment]
> You can't delete the old one, it will live "forever". We can, however, ask Google to remove the old one from the module index (I _think_, maybe I'm confusing this with removing it from pkg.go.dev).

At least we could make a PR to remove it from the main branch - and then let Google to remove it from the module index?

File: sdk/agrifood/armagfood/ci.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/agrifood/armagrifood
```
Comment (jhendrixMSFT) [inline sdk/agrifood/armagfood/ci.yml:5]: This path doesn't match the actual path.

Comment (ArcturusZhang) [reply sdk/agrifood/armagfood/ci.yml]: I just fixed this

**Review by lirenhe — APPROVED** [review]

File: sdk/agrifood/armagrifood/ci.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    include:
+    - sdk/agrifood/armagrifood
+
+pr:
+  paths:
+    include:
+    - sdk/agrifood/armagrifood
+
+stages:
+- template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml
```
Comment (weshaggard) [inline sdk/agrifood/armagrifood/ci.yml:13]:
```suggestion
- template: /eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml
```

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**